### PR TITLE
GRW-1518 / design / add country selector page design

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes
- add design to country-selector page
- [Figma doc](https://www.figma.com/file/XFYNg7y7quhTZGC9cHzEt3/Hedvig.com---MVP-Wireframes?node-id=534%3A87051&viewport=-1843%2C7064%2C0.55)

### Figma Screenie

Screenshot 2022-09-22 at 17 09 07

### Dev Screenie 🤳

Screenshot 2022-09-22 at 16 59 06

## Justify why they are needed

- 🧽

## Checklist before requesting a review

- [x] I have performed a self-review of my code

[GRW-1518]: https://hedvig.atlassian.net/browse/GRW-1518?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gustaveen — APPROVED** [review]
[nicesteve](https://app.graphite.dev/macros/HedvigInsurance)
Nice! I wonder if we should have other navigation like header and footer on that page since it wont know what market/locale to fetch it from. But let's check Peter says

**Review by alebedev — APPROVED** [review]

File: apps/store/src/components/CountrySelectorPage/CountrySelectorPage.tsx [hunk 1]
```diff
@@ -12,30 +12,39 @@ const Container = styled.div({
...
 
-const CountryOptionContainer = styled.div({
+const CountryOptionsContainer = styled.div({
   display: 'grid',
-  gap: 5,
+  gap: 8,
 })
 
+const Button = styled(LinkButton)(({ theme }) => ({
+  borderRadius: theme.space[1],
```
Comment (simonauner) [inline apps/store/src/components/CountrySelectorPage/CountrySelectorPage.tsx:24]: nit: Why should we have different border radius on this page in particular? Shouldn't we mostly stay with the defined button that we have 🤓 ? And if we want to have border-radius, shouldn't it be coded into `LinkButton`? 

The reason why I'm commenting on this is that styling our UI components with Emotion is an anti-pattern to using a Design System where buttons look and behave the same across the product 🤓

Comment (KarnellSchultz) [reply apps/store/src/components/CountrySelectorPage/CountrySelectorPage.tsx:24]: Yeah, totally agree. Thanks for that comment. I was trying to guess at what the border-radius was and reached for that styled component.

Going to go with the default and remove the styled component 👍

**Review by simonauner — APPROVED** [review]
🚀

**Pull request merged**